Adds some miscellaneous equipment to loadout

Recoherent opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds black/white gloves, the standard labcoat, trick blindfold, and a laptop to the loadout menu.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
atlas and prefab asked for more stuff and i like more stuff
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: You can now grab some gloves, labcoats, and even a transparent blindfold in loadouts. Space Radioshack has a new laptop line in stock, too, in case tablets aren't your speed.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Recoherent committed:
adds a buncha garbage wwwowow

atlas and prefab asked for these

Recoherent commented:
fun fact, there was some sort of broken loadout subtype grabber for gloves in the accessories file, which i just ASSUME was intended to cause a set of colored gloves to show up? obviously didn't work

Ms-Mee commented on code/modules/client/loadout/loadout_accessories.dm:
```diff
@@ -67,8 +67,15 @@
 	display_name = "maid neckpiece"
 	path = /obj/item/clothing/neck/maid
 
-/datum/gear/accessory/gloves
-	subtype_path = /datum/gear/accessory/gloves
```
Glove subtype definition for loading all glove subtypes, including the current evening gloves, is it not?

Ms-Mee replied:
It's there to skip the subtype during loading given it doesn't contain anything and is just there for organization, indeed.

Ms-Mee replied:
As in, it skips /datum/gear/accessory/gloves and only loads it's subtypes such as /datum/gear/accessory/gloves/evening

Ms-Mee commented on code/modules/client/loadout/loadout_accessories.dm:
```diff
@@ -67,8 +67,15 @@
+/datum/gear/accessory/whitegloves
+	display_name = "white gloves"
+	description = "Standard hand coverings for everyday use."
+	path = /obj/item/clothing/gloves/color/white
```
```suggestion
/datum/gear/accessory/gloves/black
	display_name = "black gloves"
	description = "Standard hand coverings for everyday use."
	path = /obj/item/clothing/gloves/color/black

/datum/gear/accessory/gloves/white
	display_name = "white gloves"
	description = "Standard hand coverings for everyday use."
	path = /obj/item/clothing/gloves/color/white
```

ZephyrTFA committed:
Merge branch 'master' into more-loadout-garbage

Recoherent committed:
subtypes are a thing thanks for the review

i forgot this was a button i could press so i didn't have to go manually fix it on my local. oops. would have pressed earlier

Co-authored-by: Ms-Mee <88832933+Ms-Mee@users.noreply.github.com>

MarkSuckerberg committed:
Merge branch 'master' into more-loadout-garbage
